### DISCUSSION TITLE
How to make a backup of all Spoke content from a hubs cloud

### DISCUSSION THREAD

**Discussion by wmurphyrd** [mozilla/hubs#3607] [Spoke Questions] [answered]
I may have to rebuild some hubs cloud stacks from scratch, but I've got valuable work saved in the spoke scenes on them. Is it possible to backup/restore all spoke content for a stack?

**Comment by robinkwilson** [comment, marked as answer]
Yes! 

Take a peek at the [Backup and Restore](https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html) docs. You have automatic backups already being made in the background (iirc, the automatic backups are rolling, so they frequently get overwritten by more current backups).

**I'd definitely recommend making a manual backup** just in case once your Spoke projects are setup. So if anything happens, you know you have a place to go back to.
